client/executor/common/src/wasm_runtime.rs

```diff
@@ -19,6 +19,42 @@
 use crate::error::Error;
 use sp_wasm_interface::Value;
 
+/// Call site.
```

Comment:
That's not a call site as discussed earlier. 

How about this? 

```suggestion
/// A method to be used to find the entrypoint when calling into the runtime
```